src/plugins/timelist/TimelistComponent.vue

```diff
@@ -384,9 +388,14 @@ export default {
         return activity;
       });
 
-      this.activityClosestToNowIndex = activityClosestToNowIndex;
-      this.firstCurrentActivityIndex = firstCurrentActivityIndex;
+      if (firstCurrentActivityIndex > -1) {
```

Comment:
what happens if both indexes are <= -1? can that happen?

Reply:
No that can't happen. We initialize to -1 and only assign if indices are 0+.